index.d.ts

```diff
@@ -53,11 +53,14 @@ export interface FieldProps {
 export class Field extends React.Component<FieldProps, any> {
 }
 
+export type OnSubmitCallback = (s: {name: string, values: {[key: string]: any}}) => void;
```

Comment:
`value: Object` takes any values but this entry expect only Object.
https://www.typescriptlang.org/docs/handbook/basic-types.html#any